packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -86,7 +86,9 @@ const chatMessageStyles: ComponentSlotStylesInput<
     props: p,
     variables: v,
     theme: {
-      siteVariables: { white },
+      siteVariables: {
+        colors: { white },
+      },
```

Comment:
Actually, should be moved to `siteVariables`

Reply:
What do you mean? We have the white in the `colors` and not directly in the `siteVariables`

Reply:
Sorry, updated comment, more it's more clear

Reply:
Got it, thanks!